feature/menu/src/main/java/com/guidal/feature/menu/about/AboutScreen.kt

```diff
@@ -0,0 +1,194 @@
+                onClick = {
+                    // TODO Change to Privacy Policy link
+                    aboutViewModel.toExternalLink("https://github.com/mantvydasdeltuva/guidal")
+                },
+                enabled = !uiState.isNavigating && uiState !is AboutUiState.Loading,
+                trailingIcon = UiModelMenuButtonIcon(
+                    imageVector = GuidalIcons.Outlined.Redirect,
+                    size = dimensionResource(com.guidal.core.ui.R.dimen.icon_size_20)
+                )
+            )
+
+            // Terms of Service
+            MenuButton(
+                label = stringResource(R.string.about_menu_button_label_tos),
+                onClick = {
+                    // TODO Change to ToS link
+                    aboutViewModel.toExternalLink("https://github.com/mantvydasdeltuva/guidal")
+                },
+                enabled = !uiState.isNavigating && uiState !is AboutUiState.Loading,
+                trailingIcon = UiModelMenuButtonIcon(
+                    imageVector = GuidalIcons.Outlined.Redirect,
+                    size = dimensionResource(com.guidal.core.ui.R.dimen.icon_size_20)
+                )
+            )
+
+            // Licenses
+            MenuButton(
+                label = stringResource(R.string.about_menu_button_label_licenses),
+                onClick = {
+                    // TODO Change to Licenses link
+                    aboutViewModel.toExternalLink("https://github.com/mantvydasdeltuva/guidal")
+                },
+                enabled = !uiState.isNavigating && uiState !is AboutUiState.Loading,
+                trailingIcon = UiModelMenuButtonIcon(
+                    imageVector = GuidalIcons.Outlined.Redirect,
+                    size = dimensionResource(com.guidal.core.ui.R.dimen.icon_size_20)
+                )
+            )
+
+            // Rate App
+            MenuButton(
+                label = stringResource(R.string.about_menu_button_label_rate_app),
+                onClick = {
+                    // TODO Change to Google Play application link
+                    aboutViewModel.toExternalLink("https://github.com/mantvydasdeltuva/guidal")
+                },
+                enabled = !uiState.isNavigating && uiState !is AboutUiState.Loading,
+                trailingIcon = UiModelMenuButtonIcon(
+                    imageVector = GuidalIcons.Outlined.Redirect,
+                    size = dimensionResource(com.guidal.core.ui.R.dimen.icon_size_20)
+                )
+            )
+
+            // Guidal Version
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(54.dp)
+                    .padding(horizontal = 28.dp),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = stringResource(R.string.about_menu_button_label_guidal_version),
```

Comment:
This resource is not necessary since it will not be changed, but it can be left like that